spec/decorators/user_decorator_spec.rb

```diff
@@ -72,6 +72,97 @@
     end
   end
 
+  describe '#pending_profile_requires_verification?' do
+    it 'returns false when no pending profile exists' do
+      profile = create(:profile)
+      user_decorator = UserDecorator.new(profile.user)
+
+      expect(user_decorator.pending_profile_requires_verification?).to eq false
+    end
+
+    it 'returns true when no active profile exists' do
```

Comment:
I just realized there is a conditional in the decorator that checks if `identity_not_verified?`, which I think this test was meant for, but the test set up seems to be testing something else. 

To test the `identity_not_verified?` conditional, I would create a profile with `active: false`, as opposed to one with a `deactivation_reason`.

To test the scenario where a user has a `pending_profile`, I would perhaps stub the decorator to receive `pending_profile` and return `true`, and then add more tests to the `#pending_profile` section to test the 2 other deactivation_reason scenarios.